overlord/snapshotstate/backend/helpers.go

```diff
@@ -133,7 +133,7 @@ func usersForUsernamesImpl(usernames []string, opts *dirs.SnapDirOptions) ([]*us
 	return users, nil
 }
 
-func allUsers(opts *dirs.SnapDirOptions) ([]*user.User, error) {
+func AllUsers(opts *dirs.SnapDirOptions) ([]*user.User, error) {
```

Comment:
instead of exporting it from here I think we should move and export this from the snap package with a proper doc comment there.